js/classes/entities/midground/enemies/yeti.js

```diff
@@ -87,20 +90,32 @@ class Yeti {
         this.removeFromWorld = true;
 
         // add a piece of bacon in the Yeti's place at bottom-center of yeti
-        const baconPos = new Vector(this.pos.x, this.pos.y + Yeti.SCALED_SIZE.y / 2);
-        GAME.addEntity(new FoodDrop(FoodDrop.BACON, baconPos));
+        const pos = this.base.getCenter();
+        GAME.addEntity(new FoodDrop(pos, FoodDrop.GIANT_MUSHROOM));
+    }
+
+    static get GROWL_THRESHOLD() {
```

Comment:
I would suggest naming this CHOOSE_GROWL or similar if it is just choosing different Growls.